templates/satspay/display.html

```diff
@@ -1,329 +1,210 @@
+              <!-- <q-item-label caption>Total amount to pay</q-item-label> -->
+            </q-item-section>
+
+            <q-item-section side>
               <q-badge color="blue">
-                <span v-text="charge.amount" class="text-subtitle2"></span> sat
+                <span v-text="charge.amount" class="text-subtitle2"></span>
+                <span>&nbsp;sats</span>
               </q-badge>
-            </div>
-          </div>
-          <div class="row items-center q-mt-sm">
-            <div class="col-4 q-pr-lg">Amount paid:</div>
-            <div class="col-8 q-pr-lg">
+            </q-item-section>
+          </q-item>
+          <q-separator spaced></q-separator>
+          <q-item dense>
+            <q-item-section>
+              <q-item-label>Amount paid:</q-item-label>
+              <!-- <q-item-label caption>Amount already paid</q-item-label> -->
+            </q-item-section>
+
+            <q-item-section side>
               <q-badge color="orange">
                 <span v-text="charge.balance" class="text-subtitle2"></span>
-                sat</q-badge
-              >
-            </div>
-          </div>
-          <div v-if="pendingFunds" class="row items-center q-mt-sm">
-            <div class="col-4 q-pr-lg">Amount pending:</div>
-            <div class="col-8 q-pr-lg">
+                <span>&nbsp;sats</span>
+              </q-badge>
+            </q-item-section>
+          </q-item>
+          <q-separator spaced v-if="pendingFunds"></q-separator>
+          <q-item v-if="pendingFunds" dense>
+            <q-item-section>
+              <q-item-label>Amount pending:</q-item-label>
+              <!-- <q-item-label caption>Amount pending on mempool</q-item-label> -->
+            </q-item-section>
+
+            <q-item-section side>
               <q-badge color="gray">
-                <span v-text="pendingFunds" class="text-subtitle2"></span> sat
+                <span class="text-subtitle2">${pendingFunds} sats</span>
               </q-badge>
-            </div>
-          </div>
-          <div class="row items-center q-mt-sm">
-            <div class="col-4 q-pr-lg">Amount due:</div>
-            <div class="col-8 q-pr-lg">
-              <q-badge v-if="charge.amount - charge.balance > 0" color="green">
-                <span
-                  v-text="charge.amount - charge.balance"
-                  class="text-subtitle2"
-                ></span>
-                sat
+            </q-item-section>
+          </q-item>
+          <q-separator spaced></q-separator>
+          <q-item dense>
+            <q-item-section>
+              <q-item-label>Amount due:</q-item-label>
+              <!-- <q-item-label caption>Amount left to be paid</q-item-label> -->
+            </q-item-section>
+
+            <q-item-section side>
+              <q-badge color="green">
+                <span class="text-subtitle2"
+                  >${charge.amount - charge.balance > 0 ? charge.amount -
```

Comment:
- is `${...}` actually ok?

Reply:
it uses `delimiter` from vue, so we don't need the `{% raw %}` and can use JS